suite-native/module-onboarding/src/components/AnalyticsConsentContent.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Stack, Switch, Text, VStack } from '@suite-native/atoms';
+import { Icon } from '@suite-common/icons';
+import { Translation, useTranslate } from '@suite-native/intl';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { AnalyticsInfoRow } from './AnalyticsInfoRow';
+
+type AnalyticsConsentContentProps = {
+    isEnabled: boolean;
+    setIsEnabled: (enabled: boolean) => void;
+};
+
+const consentInfoStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.small,
+    paddingVertical: utils.spacings.extraSmall,
+}));
+
+const analyticsConsentStyle = prepareNativeStyle(_ => ({
+    flex: 1,
+    justifyContent: 'space-between',
+    width: '100%',
+}));
+
+const consentWrapperStyle = prepareNativeStyle(utils => ({
+    maxHeight: 64,
+    paddingHorizontal: utils.spacings.medium,
+    paddingVertical: utils.spacings.small,
+    marginBottom: utils.spacings.small,
+    borderRadius: utils.spacings.medium,
+    backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation1,
+    width: '100%',
+}));
+
+export const AnalyticsConsentContent = ({
```

Comment:
What's the reason for making component of this? I prefer keeping things together if there is no intention to reuse them.

Reply:
Just makes it nicer to read the component above and removes overhead. To me, even if it's not going to be reused, I prefer to separate some parts of code. If we only separated parts of code that is reusable, then some components would get really large. There is no performance or readability downside so when I start to feel like component is getting big, for dev experience, I like to separate it.